### PR TITLE
Update introduction-windows-365-government.md

### PR DESCRIPTION
**Pull request by kmsft**
clarify GCC vs GCC High

### CODE REVIEW

File: windows-365/enterprise/introduction-windows-365-government.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ Windows 365 Government is a cloud-based service that automatically creates virtu
 
 Windows 365 Government customers are government agencies or public entities that qualify to use services hosted in GCC and GCC High. The services in the GCC and GCC High clouds must meet specific requirements and pass rigorous audit reviews before customers are allowed to use them.
 
-For GCC environments, users that access Windows 365 Government Cloud PCs have an identity in the public cloud while the underlying resources and content are secured in a regulated US government cloud.
+For GCC, users that access Windows 365 Government Cloud PCs have an identity in the Azure AD public tenant.  For GCC High, users that access Windows 365 Government Cloud PCs have an identity in the Azure AD Government tenant.  The underlying resources are isolated from the general public through a series of physical and logical controls.  For more information see [Azure Government](https://learn.microsoft.com/en-us/azure/azure-government/documentation-government-plan-identity)
```
Comment (JohanFreelancer9) [inline windows-365/enterprise/introduction-windows-365-government.md:37]:
```suggestion
For GCC, users who access Windows 365 Government Cloud PCs have an identity in the Azure AD public tenant.  For GCC High, users who access Windows 365 Government Cloud PCs have an identity in the Azure AD Government tenant.  The underlying resources are isolated from the general public through a series of physical and logical controls.  For more information, see [Azure Government](https://learn.microsoft.com/azure/azure-government/documentation-government-plan-identity).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @kmsft, kindly commit the suggested changes. Many thanks.

**Comment by ErikjeMS** [comment]
#sign-off

**Comment by kmsft** [comment]
> #sign-off

Hi Erik, the sign-off tag doesn't seem to work and result in a merge.  Can you merge this while I close and re-do the other PRs which will take more time?

**Comment by ErikjeMS** [comment]
#sign-off

**Comment by ErikjeMS** [comment]
> 

Only the author of the article can sign-off.

**Comment by ErikjeMS** [comment]
@kmsft Please recreate this PR in the private repo. Closing.

**Pull request closed without merging**